src/components/ec-modal/ec-modal.vue

```diff
@@ -205,6 +216,13 @@ $ec-modal-close-btn-fill-hover: $level-4-tech-blue !default;
 
   @include z-index-modal;
 
+  // The z-index will be on between 200 and 250 because the 250 is the z-index for the loading
+  @for $i from 200 through 250 {
```

Comment:
We are not going to generate 50 CSS classes and then rarely use them. 

We can use a `style` tag for this. Nobody is going to override this property once is set.

Or we can use the same level mechanism as with tooltips, using predefined levels.